runtime/libcn/dune

```diff
@@ -5,7 +5,7 @@
        (:src (glob_files src/*.c)))
  (action
   (progn
-   (run cc -Iinclude/ -c %{src})
+   (run cc -Iinclude/ -c -g %{src})
```

Comment:
Do we always want debug information for the runtime library itself? I feel like it's sufficient to just compile the CN-annotated files with `-g`

Reply:
Yes - you often want to set the break point at `cn_exit_aux` and you can't do that with debug info.